ui/src/frame.tsx

```diff
@@ -32,7 +32,12 @@ const
     },
     body: {
       flexGrow: 1,
+      position: 'relative',
     },
+    frame: {
+      position: 'absolute',
+      top: 10, left: 0, right: 0, bottom: 0
```

Comment:
The 10px margin seems odd. The inter-component spacing should take care of this, no?

Reply:
Yes, this was a typo.